packages/iris-grid/src/mousehandlers/IrisGridDataSelectMouseHandler.ts

```diff
@@ -12,7 +12,7 @@ import type IrisGrid from '../IrisGrid';
  */
 class IrisGridDataSelectMouseHandler extends GridMouseHandler {
   constructor(irisGrid: IrisGrid) {
-    super();
+    super(825);
```

Comment:
This could probably use a comment as to what `825` is

Reply:
ah, I see it is `order` now in the base class.

Reply:
Yea, we could (and probably should) have constants for a few different levels of this... https://github.com/deephaven/web-client-ui/issues/1859